packages/front-end/pages/power-calculator/index.tsx

```diff
@@ -115,6 +115,16 @@ const PowerCalculationPage = (): React.ReactElement => {
         <PowerCalculationSettingsModal
           close={() => setShowModal(false)}
           onSuccess={(p) => {
+            track("power-calculation-settings-update", {
+              numberOfMetrics: p.metrics.length,
+              metricsMetaData: Object.keys(p.metrics).map((m: string) => {
```

Comment:
Why loop over the keys and then call the object with p.metrics? I just don't know why that's the preferred approach.

In any case, looks good, we just need to ensure we can add `metricsMetaData` to our whitelist.

Reply:
p.metrics is an object of objects so the first is finding the object key `object.keys` the `map` is just going over the metric object to get the info we want out